schema/policy.go

```diff
@@ -34,6 +35,8 @@ const (
 	ActionUpdate = "update"
 	// ActionDelete allows to delete a resource
 	ActionDelete = "delete"
+	// ActionAttach allows a resource to have a relation to another resource
+	ActionAttach = "attach"
```

Comment:
I'm afraid of potential custom action conflict - could we \_\_attach\_\_ instead?

Reply:
Changed "attach" to "__attach__".